server/stream.go

```diff
@@ -97,7 +97,9 @@ type ClusterInfo struct {
 type PeerInfo struct {
 	Name    string        `json:"name"`
 	Current bool          `json:"current"`
+	Offline bool          `json:"offline,omitempty"`
 	Active  time.Duration `json:"active"`
+	Lag     uint64        `json:"lag,omitempty"`
```

Comment:
@derekcollison would you say "How many uncomitted operations this peer is behind the leader" accurately describes this?

Reply:
Yes correct, not necessarily things measured by the FSM.